DArray/pagerank.jl

```diff
@@ -1,55 +1,80 @@
-module Pagerank
+module PageRank
+module DArray
```

Comment:
How about calling this differently so that we don't have to use `DistributedArrays.DArray` everywhere else. I don't have a good idea for a name, but maybe we should suffix the modules with `Benchmark` or `Bench`, because we will run into this again with `MPI` and maybe `Dagger`.